save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/diktat/DemoAdditionalParams.kt

```diff
@@ -14,6 +14,109 @@ import kotlinx.serialization.Serializable
 data class DemoAdditionalParams(
     val mode: DiktatDemoMode = DiktatDemoMode.WARN,
     val tool: DiktatDemoTool = DiktatDemoTool.DIKTAT,
-    val config: String? = null,
-    val language: Languages = Languages.KOTLIN
-) : DemoAdditionalParams
+    val config: List<String> = defaultDiktatConfig,
+    val language: Languages = Languages.KOTLIN,
+) : DemoAdditionalParams {
+    companion object {
+        val defaultDiktatConfig = """
```

Comment:
```suggestion
        @Language("YAML")
        val defaultDiktatConfig = """
```

Bacisally, both `YaMl` and `yMl` (in any case) work.

Reply:
sure, thanks

Reply:
Seems that commonMain does not have `@Language` annotation so will remove as was

Reply:
https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments